cypress/page/login.page.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable require-jsdoc */
+class LoginPage {
+  private email: string;
+  private psw: string;
+  private signInBtn: string;
+
+  constructor() {
+    this.email = "#email";
+    this.psw = "#passwd";
+    this.signInBtn = "#SubmitLogin > span";
+  }
+
+  public login(): void {
```

Comment:
Try to receive the email and password as parameters. That way, it will not be always the same, but is the test that will decide which user and password use when calling the login